workflows/kallisto/lcdb-workflows.bashrc

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+if [ -n "$SLURM_JOBID" ]; then
+    export TMPDIR="/lscratch/$SLURM_JOBID"
+fi
```

Comment:
It may be good to set pipefail too. 

Do you use umask/group permissions much?